README.md

```diff
@@ -1,5 +1,11 @@
 # Accessible autocomplete
 
+The accessible autocomplete is a component that predicts the rest of a word or string a user is typing.
+
+Do not use the accessible autocomplete for services where users have to declare their country or territory.
```

Comment:
Not sure 'declare' is the right verb, but I thought 'select' might be inaccurate because some users could type out the word in full. Also didn't want to write 'state' in case anyone reads it as a noun and not a verb, or gets confused by seeing 'state' and 'country' in the same sentence...

Reply:
Provide?